src/ensembl/utils/argparse.py

```diff
@@ -312,17 +311,23 @@ def parse_args(self, *args: Any, **kwargs: Any) -> argparse.Namespace:  # type:
         """
         arguments = super().parse_args(*args, **kwargs)
         # Build and add an sqlalchemy.engine.URL object for every server group added
-        for prefix in self.__server_groups:
+        pattern = re.compile(r"([\w-]*)host$")
```

Comment:
what pattern input is this searching for hits on `host server details` input param?

Reply:
So this is already accessing the arguments passed to the command, i.e. `--host`, `--user`, etc. (after resolving `$(server details script)`), and since there can be many server "sets" with prefixes (e.g. `--src_host` and `--dst_host`), we need to make sure all of them are found.

Reply:
I wasn't sure if you were pattern matching across various ENS mysql hosts etc. My bad thanks for clearing my confusion